### DISCUSSION TITLE
Nixos support?

### DISCUSSION THREAD

**Discussion by tmpm697** [folke/noice.nvim#573] [General]
Currently I can't make this plugin work under nixos (plugin load up but nothing happens -- no `Noice` command avail) and this plugin is core plugin for me :(

Just tried on latest HEAD and NVIM v0.10.0-dev-33e1a8c

Anyone has a working setup for noice.nvim under nixos? Thanks.

**Comment by tmpm697** [comment]
If use lazy.nvim, it's loaded --> this plugin not loaded use native nvim way to load plugins.

EDIT: due to how my config is loaded causes this issue.

**Comment by siph** [comment]
I use [nixvim](https://github.com/nix-community/nixvim) for my config and `Noice` works great with it. You might check their code base for some clues.
